tests/test_hubspot_bookmarks.py

```diff
@@ -65,9 +67,13 @@ def create_test_data(self, expected_streams):
                 self.expected_records['email_events'] += email_records
             else:
                 # create records, one will be updated between syncs
+                # create one static list and the rest dynamic list
                 for _ in range(self.times):
                     record = self.test_client.create(stream)
                     self.expected_records[stream] += record
+                if stream in 'contact_lists':
```

Comment:
I'm not sure this is doing what you expect 
```suggestion
                if stream == 'contact_lists':
```
because you are comparing stream to a string with `in` it will only be true if stream is one of the characters in the string.  so if stream = `c` it would be true.  In is generally used with a list or tuple.